includes/class-micropub-endpoint.php

```diff
@@ -437,6 +437,13 @@ private static function create( $user_id ) {
 			}
 		}
 		$args = static::mp_to_wp( static::$input );
+
+		// Allow Filtering of Post Type
+		$args['post_type'] = apply_filters( 'micropub_post_type', 'post', static::$input );
```

Comment:
is this really correct? shouldn't it be something like

```$args['post_type'] = apply_filters( 'micropub_post_type', 'post', static::$input['post_type'] );```

???

Reply:
ah sorry, misread it...

`post` is the param and `static::$input` is only the context.

Reply:
Yes, the request was for a way to set a custom post type or a custom taxonomy based on input parameters. So two specific filters.